Cargo.toml

```diff
@@ -44,3 +43,6 @@ rand = "0.8.4"
 
 [package.metadata.docs.rs]
 features = ["dispatch"]
+
+[patch.crates-io]
+trussed = { git = "https://github.com/robin-nitrokey/trussed", branch = "littlefs2" }
```

Comment:
This will have to be changed after merging https://github.com/trussed-dev/trussed/pull/107 before merging this.